great_expectations/data_context/util.py

```diff
@@ -17,6 +18,41 @@
 logger = logging.getLogger(__name__)
 
 
+def safe_mmkdir(directory, exist_ok=True):
+    """Simple wrapper since exist_ok is not available in python 2"""
+    if not isinstance(directory, six.string_types):
+        raise TypeError("directory must be of type str, not {0}".format({
+            "directory_type": str(type(directory))
+        }))
+
+    if not exist_ok:
+        raise ValueError(
+            "This wrapper should only be used for exist_ok=True; it is designed to make porting easier later")
+    try:
+        os.makedirs(directory)
+    except OSError as e:
+        if e.errno != errno.EEXIST:
+            raise
+
+
+def safe_rrmdir(directory, exist_ok=True):
+    if not isinstance(directory, six.string_types):
+        raise TypeError("directory must be of type str, not {0}".format({
+            "directory_type": str(type(directory))
+        }))
+
+    if not exist_ok:
+        raise ValueError(
+            "This wrapper should only be used for exist_ok=True; it is designed to make porting easier later")
+    try:
+        if os.path.exists(directory):
+            shutil.rmtree(directory)
+        #os.removedirs(directory)
+    except OSError as e:
+        if e.errno != errno.EEXIST:
+            raise
+
+
```

Comment:
Remove

Reply:
Yes, develop merge conflict

Reply:
We were pair walking though this. Resolved.